iron-a11y-keys-behavior.html

```diff
@@ -411,14 +374,21 @@
           event.stopPropagation();
         }
 
-        keyBindings.forEach(function(keyBinding) {
-          var keyCombo = keyBinding[0];
-          var handlerName = keyBinding[1];
+        // if event has been already prevented, don't do anything
+        if (event.defaultPrevented) {
+          return;
+        }
 
-          if (!event.defaultPrevented && keyComboMatchesEvent(keyCombo, event)) {
+        var eventKey = normalizedKeyForEvent(event);
+        for (var i = 0; i < keyBindings.length; i++) {
+          var keyCombo = keyBindings[i][0];
+          var handlerName = keyBindings[i][1];
+          if (keyComboMatchesEvent(keyCombo, event, eventKey)) {
             this._triggerKeyHandler(keyCombo, handlerName, event);
+            // found the matching keycombo, we can exit the loop
+            return;
```

Comment:
I see your points. I believe that my proposal of supporting multiple listeners doesn't really help in neither of the 2 cases. The main issue is in the direct "return" in this line.
We should instead check if the event default was prevented, and only in that case return.
I'll update this, and write some unit tests that make it clear what's the intent.
Also, I'll be sorting the keyBindings to give precedence to combos that require modifiers, so they'll be checked first.